main-site/components/mailinglist-signup/MailinglistSignup.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import addToMailChimp from 'gatsby-plugin-mailchimp';
+import { StyledEmailInput } from '../../../shared-ui/components/footer/Footer.styles';
+import PrimaryButton from '../../../shared-ui/components/primary-button/PrimaryButton';
+import { StyledPrimaryButton } from '../../../shared-ui/components/primary-button/PrimaryButton.styles';
```

Comment:
Is there a reason you're using StyledPrimaryButton instead of the PrimaryButton itself?